facade/src/main/resources/buildConfigurationSupportedGenericParameters.properties

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-CUSTOM_PME_PARAMETERS=Additional parameters, which will be passed to the PME CLI executable during the alignment before the build. The format is as you would enter them on a command line, and each MUST start with a dash.
+ALIGNMENT_PARAMETERS=Additional parameters, which will be passed to the PME CLI executable during the alignment before the build. The format is as you would enter them on a command line, and each MUST start with a dash.
 BREW_BUILD_NAME=Specify the Brew build name of the build configuration. This is used to override the default value, and can be useful for builds that disable PME. For Maven builds the format should be '<groupid>:<artifactid>'.
 CUSTOM_PROJECT_MANIPULATOR_PARAMETERS=Additional parameters, which will be passed to the Project Manipulator CLI executable during the alignment (NPM, etc, non-Maven projects) before the build. The format is as you would enter them on a command line, and each MUST start with a dash.
```

Comment:
CUSTOM_PROJECT_MANIPULATOR_PARAMETERS shopuld have been removed as ALIGNMENT_PARAMETERS replaces it, right?

Reply:
Correct. I missed. Thanks @pkocandr 
@dominikbrazdil Please create a new PR to fix it.